python/run_plugin_test.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash -e
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+pip install pyspark~=4.0
+pushd ../jvm
+mvn clean test
+popd
+pip install -r requirements_dev.txt
```

Comment:
Is this still needed?

Reply:
This is to restore the spark version to 3.4 for the remaining tests.